app/models/partnerships/iaa_gtc.rb

```diff
@@ -0,0 +1,14 @@
+class Partnerships::IaaGtc < ApplicationRecord
+  self.table_name = 'iaa_gtcs'
+
+  belongs_to :partner_account
+  belongs_to :iaa_status
+  has_many :iaa_orders
+
+  validates :gtc_number, presence: true, uniqueness: true
+  validates :mod_number, presence: true,
+                         numericality: { only_integer: true,
+                                         greater_than_or_equal_to: 0 }
+  # validates :partner_account, foreign_key: true
```

Comment:
The namespacing of the models is breaking my very naive [validator](https://github.com/18F/identity-idp-config/blob/oyk-partnerships-data/lib/foreign_key_validator.rb) - I'm wondering if we even need ActiveModel validations in the IdP since the foreign key constraints in the DB will catch any issues and I only added the validator in the config repo to make the YAML validation easier to deal with. Curious what you think!

Reply:
Interesting point. However, I hope these tables only exist in the IDP some day. I think it makes sense to add the ActiveModel validations and tests here as well for uniformity.